### DISCUSSION TITLE
유니티 맵 테스트는 어떻게 하나요??

### DISCUSSION THREAD

**Discussion by yj098** [naverz/zepeto-studio-kor#975] [5. Q&A]
![image](https://user-images.githubusercontent.com/99742984/177714143-297f595f-a097-4eaf-b875-9ae5abb7534c.png)

복사한 url은 어디에서 접속해야 테스트 해 볼 수 있을까요?? ㅠ

**Comment by YSHFM** [comment]
모바일로 해당 링크 접속하면 제페토 앱 켜지면서 해당 맵 월드로 실행이 됩니다~

**Reply by yj098** [reply]
감사합니다 :)
